### DISCUSSION TITLE
New Student Developer, Query about benefits….

### DISCUSSION THREAD

**Discussion by The-MarkIV** [community/community#136384] [GitHub Education]
### Hi everyone,

Hello, I am new here and I have recently been approved for Student Developer Pack and I saw that it was written that if I don’t receive the benefits within three days then i should raise a conversation …. What should I do?

**Comment by davevad93** [comment]
Hi @Walker087476 ,

The [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) state that "***If you have not received your education benefits within 5 days of your approval status to raise a discussion on the [GitHub Education community](https://github.com/orgs/community/discussions/categories/github-education) and the Education community team will be able to assist in resolving this for you***"

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

Note that the waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks; 

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @Walker087476 was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849

**Reply by The-MarkIV** [reply]
Thankyou sir

**Reply by davevad93** [reply]
You're welcome @Walker087476 ! 👍
